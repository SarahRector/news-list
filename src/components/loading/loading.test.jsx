import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Loading from './loading';

describe('Loading component', () => {
  afterEach(() => cleanup());
  it('renders Loading Screen', () => {
    const { asFragment } = render(<Loading />);
    expect(asFragment()).toMatchSnapshot();
  });
});
