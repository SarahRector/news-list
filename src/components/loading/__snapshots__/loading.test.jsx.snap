// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component renders Loading Screen 1`] = `
<DocumentFragment>
  <img
    alt="Helga walking while screen loads"
    class="Loading"
    src="https://media.giphy.com/media/26Ff1nps1k57NTH2w/giphy.gif"
  />
</DocumentFragment>
`;
